scripts/validate_container.sh

```diff
@@ -181,7 +181,7 @@ do
             # Skipped long-running tests (variational optimization loops) for the "remote-mqpu" target to keep CI runtime managable.
             # A simplified test for these use cases is included in the 'test/Remote-Sim/' test suite. 
             # Skipped tests that require passing kernel callables to entry-point kernels for the "remote-mqpu" target.
-            if [[ "$ex" == *"vqe_h2"* || "$ex" == *"qaoa_maxcut"* || "$ex" == *"gradients"* || "$ex" == *"grover"* || "$ex" == *"multi_controlled_operations"* || "$ex" == *"phase_estimation"* || "$ex" == *"trotter_kernel_mode"* || "$ex" == *"builder.cpp"* ]];
+            if [[ "$ex" == *"vqe_h2"* || "$ex" == *"qaoa_maxcut"* || "$ex" == *"gradients"* || "$ex" == *"grover"* || "$ex" == *"multi_controlled_operations"* || "$ex" == *"phase_estimation"* || "$ex" == *"building_kernels"* || "$ex" == *"trotter_kernel_mode"* || "$ex" == *"builder.cpp"* ]];
```

Comment:
Oops, I meant to put the comment here: Could you please add a link to https://github.com/NVIDIA/cuda-quantum/issues/2299 in this file itself to explain that is the reason for this test being disabled? (We should probably do that for the others too, but those can probably wait.)

Reply:
On second thought, I think these changes should be reverted because I think the required changes would be near here instead: https://github.com/NVIDIA/cuda-quantum/blob/94f6ee513565549bd6b8d5f909cc64a46130cab0/.github/workflows/integration_tests.yml#L670.